pootle/core/initdb.py

```diff
@@ -244,15 +253,16 @@ def create_default_projects(self):
         """
         en = self.require_english()
 
+        po = Format.objects.get(name="po")
```

Comment:
in this case i think its better to set the local var explicitly at the top - as we do for `en` as its reused in the method